aiida_wannier90/calculations.py

```diff
@@ -466,3 +411,119 @@ def _validate_input_parameters(self, parameters):
                 'The following blocked keys were found in the parameters: {}'.
                 format(", ".join(existing_blocked_keys))
             )
+
+    def _get_input_file_lists(self, pp_setup):
+        """
+        Generate the lists of files to copy and link from the
+        'local_input_folder' and 'remote_input_folder'.
+        """
+        if pp_setup:
+            return _InputFileLists([], [], [])
+
+        input_file_specs = [
+            _InputFileSpec(suffix=suffix, required=True, always_copy=False)
+            for suffix in ['.mmn', '.amn']
+        ] + [
+            _InputFileSpec(suffix=suffix, required=False, always_copy=False)
+            for suffix in [
+                '.eig', '.spn', '.uHu', '_htB.dat', '_htL.dat', '_htR.dat',
+                '_htC.dat', '_htLC.dat', '_htCR.dat', '.unkg'
+            ]
+        ] + [_InputFileSpec(suffix='.chk', required=False, always_copy=True)]
+
+        optional_file_globs = ['UNK*']
+
+        # #NOTE: remote_input_folder -> parent_calc_folder (for consistency)
+        if 'remote_input_folder' in self.inputs:
+            return self._get_remote_input_file_lists(
+                input_file_specs=input_file_specs,
+                optional_file_globs=optional_file_globs
+            )
+        return self._get_local_input_file_lists(
+            input_file_specs=input_file_specs,
+            optional_file_globs=optional_file_globs
+        )
+
+    def _get_remote_input_file_lists(
+        self, input_file_specs, optional_file_globs
+    ):
+        """
+        Generate the lists of input files for the case of a remote input folder.
```

Comment:
Maybe we should add a comment here explaining why for remote input files we use blobbing (i.e., that in AiiDA if we don't use globbing and the file is not there in the remote, AiiDA will raise an error) - maybe with a link to an appropriate issue on aiida-core?

Reply:
Done!